airgun/views/repository.py

```diff
@@ -206,6 +225,25 @@ class YumRepository(View):
         class NoSpecificHttpProxy(View):
             pass
 
+    @repo_content.register('ansible collection')
+    class AnsibleCollectionRepository(View):
+        arch_restrict = EditableEntrySelect(name='Restrict to architecture')
+        upstream_url = EditableEntry(name='Upstream URL')
+        requirements = EditableEntry(name='Requirements')
+        verify_ssl = EditableEntryCheckbox(name='Verify SSL')
+        upstream_authorization = AuthorizationEntry(name='Upstream Authorization')
+        metadata_type = EditableEntrySelect(name='Yum Metadata Checksum')
```

Comment:
```suggestion

```

Reply:
@pondrejk metadata_type is not needed here right?

Reply:
@jameerpathan111 fixed, thanks!